Public/Src/Engine/Dll/Tracing/Log.cs

```diff
@@ -1085,6 +1085,15 @@ public abstract partial class Logger
             Message = "/unsafe_UnexpectedFileAccessesAreErrors disabled: Invalid file accesses by tools are configured to trigger warnings instead of errors. This is an unsafe (diagnostic only) configuration since it removes all guarantees of build correctness.")]
         public abstract void ConfigUnsafeUnexpectedFileAccessesAsWarnings(LoggingContext context);
 
+        [GeneratedEvent(
+            (ushort)EventId.ConfigUnsafeIgnoreUndeclaredAccessesUnderSharedOpaques,
+            EventGenerators = EventGenerators.LocalOnly,
+            EventLevel = Level.Warning,
+            Keywords = (int)Keywords.UserMessage,
+            EventTask = (int)Tasks.Engine,
+            Message = "/unsafe_IgnoreUndeclaredAccessesUnderSharedOpaques enabled: Undeclared accesses under shared opaques will not be reported. This is an unsafe configuration since it removes all guarantees of build correctness.")]

```

Comment:
"Undeclared accesses under shared opaques will not be reported" sounds very counter-intuitive to me, because the main idea behind opaque directories is exactly to not have to declare any accesses.

Reply:
In terms of outputs, yes. In terms of inputs you are still only allowed to read what you declare. I could rephrase it as 'undeclared reads' if that sounds better.

Reply:
maybe it does sound a bit better, but I'm not insisting